src/main/java/ch/njol/skript/expressions/ExprDamage.java

```diff
@@ -39,47 +41,71 @@
-@Examples({"increase the damage by 2"})
+@Description({
+	"How much damage is done in a damage event, possibly ignoring armour, criticals and/or enchantments.",
+	"Can be changed (remember that in Skript '1' is one full heart, not half a heart)."
+})
+@Examples({
+	"on damage of player:",
+		"\tdamage cause was lightning",
+		"\tvictim is wearing a leather helmet",
+		"\tsubtract 2.5 from the damage",
+		"\tmessage \"The smite was lessened by your leather helmet!\" to the victim"
+})
 @Since("1.3.5")
+@RequiredPlugins("Spigot 1.14+ for the item damage event.")
 @Events("damage")
 public class ExprDamage extends SimpleExpression<Number> {
-	
+
+	private final static boolean ITEM_DAMAGE = Skript.classExists("org.bukkit.event.player.PlayerItemDamageEvent");
+
 	static {
 		Skript.registerExpression(ExprDamage.class, Number.class, ExpressionType.SIMPLE, "[the] damage");
 	}
-	
-	@SuppressWarnings("null")
+
 	private Kleenean delay;
-	
+
 	@Override
-	public boolean init(final Expression<?>[] exprs, final int matchedPattern, final Kleenean isDelayed, final ParseResult parseResult) {
-		if (!getParser().isCurrentEvent(EntityDamageEvent.class, VehicleDamageEvent.class)) {
-			Skript.error("The expression 'damage' may only be used in damage events", ErrorQuality.SEMANTIC_ERROR);
+	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
+		if (ITEM_DAMAGE) {
+			if (!getParser().isCurrentEvent(EntityDamageEvent.class, VehicleDamageEvent.class, PlayerItemDamageEvent.class)) {
+				Skript.error("The expression 'damage' may only be used in damage events");
+				return false;
+			}
+		} else if (!getParser().isCurrentEvent(EntityDamageEvent.class, VehicleDamageEvent.class)) {
+			Skript.error("The expression 'damage' may only be used in damage events");
 			return false;
 		}
 		delay = isDelayed;
 		return true;
 	}
-	
+
 	@Override
 	@Nullable
-	protected Number[] get(final Event e) {
-		if (!(e instanceof EntityDamageEvent || e instanceof VehicleDamageEvent))
+	protected Number[] get(Event event) {
+		if (!(event instanceof EntityDamageEvent || event instanceof VehicleDamageEvent))
 			return new Number[0];
-		
-		if (e instanceof VehicleDamageEvent)
-			return CollectionUtils.array(((VehicleDamageEvent) e).getDamage());
-		return CollectionUtils.array(HealthUtils.getDamage((EntityDamageEvent) e));
-	}
+
+		if (event instanceof VehicleDamageEvent)
+			return CollectionUtils.array(((VehicleDamageEvent) event).getDamage());
+
+		if (event instanceof EntityDamageEvent)
+			return CollectionUtils.array(HealthUtils.getDamage((EntityDamageEvent) event));
 	
+		if (ITEM_DAMAGE) {
+			if (!(event instanceof PlayerItemDamageEvent))
+				return new Number[0];
+			return CollectionUtils.array(((PlayerItemDamageEvent) event).getDamage());
```

Comment:
I don't think this is needed, it's handled below it.
```suggestion
		if (ITEM_DAMAGE && event instanceof PlayerItemDamageEvent) {
			return CollectionUtils.array(((PlayerItemDamageEvent) event).getDamage());
```